### PR TITLE
Add CHANGELOG release notes for 1.4.1

### LINKED ISSUE #2142: CHANGELOG release notes for 1.4.1
- Fixed missing "Share" option under menu on Android Tablet. ([#2121](https://github.com/brave/browser-android-tabs/issues/2121)
- Fixed "Not enough tokens" message shown while tipping due to wallet balance being rounded up. ([#2079](https://github.com/brave/browser-android-tabs/issues/2079))
- Fixed an issue where rewards grant is claimed but not confirmed on rewards panel. ([#2074](https://github.com/brave/browser-android-tabs/issues/2074))
- Fixed rewards summary items being wrongly categorized under rewards panel. ([#2082](https://github.com/brave/browser-android-tabs/issues/2082))
- Fixed Adblock/Tracking protection/HTTPS Everywhere on Chromium 77 upgrade. ([#2084](https://github.com/brave/browser-android-tabs/issues/2084))
- Fixed headers for partner sites on Chromium 77 upgrade. ([#2085](https://github.com/brave/browser-android-tabs/issues/2085))
- Fixed crash when deleting browser data. ([#2090](https://github.com/brave/browser-android-tabs/issues/2090))
- Upgrade to Chromium 77.0.3865.116. ([#2136](https://github.com/brave/browser-android-tabs/issues/2136))

### PR DESCRIPTION
**Pull request by srirambv**
Fixes #2142

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [1.4.1](https://github.com/brave/browser-android-tabs/releases/tag/v1.4.1)
+
+ - Fixed missing "Share" option under menu on Android Tablet. ([#2121](https://github.com/brave/browser-android-tabs/issues/2121)
+ - Fixed "Not enough tokens" message shown while tipping due to wallet balance being rounded up. ([#2079](https://github.com/brave/browser-android-tabs/issues/2079))
+ - Fixed an issue where rewards grant is claimed but not confirmed on rewards panel. ([#2074](https://github.com/brave/browser-android-tabs/issues/2074))
+ - Fixed rewards summary items being wrongly categorized under rewards panel. ([#2082](https://github.com/brave/browser-android-tabs/issues/2082))
+ - Fixed Adblock/Tracking protection/HTTPS Everywhere on Chromium 77 upgrade. ([#2084](https://github.com/brave/browser-android-tabs/issues/2084))
+ - Fixed headers for partner sites on Chromium 77 upgrade. ([#2085](https://github.com/brave/browser-android-tabs/issues/2085))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Was this issue present before or did it just occur when we upgraded to C77? We should exclude if users haven't seen this before.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [1.4.1](https://github.com/brave/browser-android-tabs/releases/tag/v1.4.1)
+
+ - Fixed missing "Share" option under menu on Android Tablet. ([#2121](https://github.com/brave/browser-android-tabs/issues/2121)
+ - Fixed "Not enough tokens" message shown while tipping due to wallet balance being rounded up. ([#2079](https://github.com/brave/browser-android-tabs/issues/2079))
+ - Fixed an issue where rewards grant is claimed but not confirmed on rewards panel. ([#2074](https://github.com/brave/browser-android-tabs/issues/2074))
+ - Fixed rewards summary items being wrongly categorized under rewards panel. ([#2082](https://github.com/brave/browser-android-tabs/issues/2082))
+ - Fixed Adblock/Tracking protection/HTTPS Everywhere on Chromium 77 upgrade. ([#2084](https://github.com/brave/browser-android-tabs/issues/2084))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Was this issue present before or did it just occur when we upgraded to C77? We should exclude if users haven't seen this before.

Comment (srirambv) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 1 above)
Comment (srirambv) [reply CHANGELOG.md]: resolved

**Comment by srirambv** [comment]
@kjozwiak  fixed. ready for re-review

**Pull request merged**